src/Symfony/Component/String/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/string",
+    "type": "library",
+    "description": "Symfony String component",
+    "keywords": ["string", "utf8", "utf-8", "grapheme", "i18n"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "symfony/polyfill-intl-grapheme": "~1.0",
```

Comment:
I don't know the standards symfony uses for the dependencies, but since this is a new package, why not require the latest versions of these polyfills? (1.12 i think)

Reply:
We prefer keeping version ranges as wide as possible, to impose as little as possible to dependents.